spec/integration/data_stream/fields/fields.spec.yml

```diff
@@ -6,7 +6,17 @@ spec:
   type: array
   items:
     type: object
-    additionalProperties: true
+    additionalProperties: false
+    patternProperties:
+      # Soft validation on some properties that can be present but not validated or used yet.
```

Comment:
The main reason I am leaving it here is to reduce the number of changes that package developers should make to migrate to v2, and to avoid conflicts with fields copied from ECS. But these fields are not really used.

Reply:
Ok, if it is not required too many changed done by developer to migrate, it will be easier that packages get updated to 2.0.0 :+1: 
Those could be removed in a following major if it is required.